Cargo.toml

```diff
@@ -40,7 +40,7 @@ keywords = ["zero-copy", "communication", "ipc", "publish-subscribe", "request-r
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/eclipse-iceoryx/iceoryx2"
-rust-version = "1.75"
+rust-version = "1.78"
```

Comment:
But the internal dependency `cargo-platform` requires 1.78. It is pulled as a requirement from bindgen I think. This will be a hack to extract this. Who are the MSRV users? And why do they need 1.75?